test/e2e/kubernetes/kubernetes.go

```diff
@@ -214,19 +214,26 @@ func CreateNginxPodInNode(ctx context.Context, k8s *kubernetes.Clientset, nodeNa
 		},
 	}
 
+	return CreateNginxPod(ctx, k8s, pod, logger)
+}
+
+func CreateNginxPod(ctx context.Context, k8s *kubernetes.Clientset, pod *corev1.Pod, logger logr.Logger) error {
```

Comment:
Should this be named `CreatePod`? there is nothing particular about nginx in this method right?